### ISSUE TITLE
ImportError: No module named '_gdal_array' on py3

### ISSUE THREAD

**Issue by fmaussion** [conda-forge/gdal-feedstock#131]
Hi, 

I don't know when it started but I think it is recent. It happens only on py3, on py2 there is no apparent problem:

```
/home/docs/checkouts/readthedocs.org/user_builds/oggm/conda/docs/lib/python3.5/site-packages/osgeo/gdalnumeric.py in <module>()
----> 1 from osgeo.gdal_array import *
      2 from numpy import *

/home/docs/checkouts/readthedocs.org/user_builds/oggm/conda/docs/lib/python3.5/site-packages/osgeo/gdal_array.py in <module>()
    166 
    167 import numpy
--> 168 import _gdal_array
    169 
    170 from . import gdalconst

ImportError: No module named '_gdal_array'
```

Thanks!

**Comment by fmaussion** [comment]
Uh, it occurs to me that this is maybe a relative import problem in gdal, not in conda-forge?

**Comment by kmuehlbauer** [comment]
@fmaussion Can you post the full trace?

**Comment by fmaussion** [comment]
Yes sorry, the relevant part starts here:

```
/home/docs/checkouts/readthedocs.org/user_builds/oggm/conda/docs/lib/python3.5/site-packages/oggm-0.1.1.dev0-py3.5.egg/oggm/core/preprocessing/gis.py in glacier_masks(gdir)
    486     # open srtm tif-file:
    487     dem_ds = gdal.Open(gdir.get_filepath('dem'))
--> 488     dem = dem_ds.ReadAsArray().astype(float)
    489 
    490     # Correct the DEM (ASTER...)

/home/docs/checkouts/readthedocs.org/user_builds/oggm/conda/docs/lib/python3.5/site-packages/osgeo/gdal.py in ReadAsArray(self, xoff, yoff, xsize, ysize, buf_obj, buf_xsize, buf_ysize, buf_type, resample_alg, callback, callback_data)
   1971         parameters should generally not be specified if buf_obj is specified. The array is returned"""
   1972 
-> 1973         from . import gdalnumeric
   1974         return gdalnumeric.DatasetReadAsArray( self, xoff, yoff, xsize, ysize, buf_obj,
   1975                                                buf_xsize, buf_ysize, buf_type,

/home/docs/checkouts/readthedocs.org/user_builds/oggm/conda/docs/lib/python3.5/site-packages/osgeo/gdalnumeric.py in <module>()
----> 1 from osgeo.gdal_array import *
      2 from numpy import *

/home/docs/checkouts/readthedocs.org/user_builds/oggm/conda/docs/lib/python3.5/site-packages/osgeo/gdal_array.py in <module>()
    166 
    167 import numpy
--> 168 import _gdal_array
    169 
    170 from . import gdalconst

ImportError: No module named '_gdal_array'
```

For the full reports see for example our [appveyor build](https://ci.appveyor.com/project/fmaussion/oggm/build/1.0.454)

**Comment by ocefpaf** [comment]
I can reproduce that. It seems related to #129. Rolling back the build number fixes it.

```shell
> conda install gdal=2.1.2=np111py35_2
python -c 'from osgeo.gdal_array import *'
```

I am checking how to fix this while preserving #129 changes.
